aiochsa/pool.py

```diff
@@ -55,7 +55,7 @@ def __init__(
         self._session = session_class(
             timeout=ClientTimeout(**timeout_params)
         )
-        params.update(dsn_to_params(dsn))
+        params = dsn_to_params(dsn) | params
```

Comment:
This way it won't work in older Python. There are still active users of aiochsa with Python 3.7 I have to take into account.